wordpress/wp-content/plugins/memberful-wp/src/paywall/renderer.php

```
@@ -0,0 +1,346 @@
+
+	/**
+	 * Whether the builder paywall is the active rendering mode.
+	 *
+	 * @return bool
+	 */
+	private static function is_builder_mode(): bool {
+		$config = Memberful_Paywall_Config::get();
+
+		return 'builder' === $config['mode'];
+	}
+}
```

PHPMD (2.15.0)

[error] The class Memberful_Paywall_Renderer is not named in CamelCase. (undefined)

(CamelCaseClassName)